packages/appium/docs/mkdocs-en.yml

```diff
@@ -12,50 +12,50 @@ theme:
 nav:
   - Welcome: index.md
   - Introduction:
-    - Intro to Appium:
-      - Overview of Appium: intro/index.md
-      - intro/drivers.md
-      - intro/clients.md
-    - intro/requirements.md
-    - intro/history.md
+      - Intro to Appium:
+          - Overview of Appium: intro/index.md
+          - intro/drivers.md
+          - intro/clients.md
+      - intro/requirements.md
+      - intro/history.md
   - Quickstart:
-    - quickstart/index.md
-    - quickstart/install.md
-    - quickstart/uiauto2-driver.md
-    - Write a Test:
-      - quickstart/test-js.md
-      - quickstart/test-py.md
-      - quickstart/test-java.md
-      - quickstart/test-rb.md
-    - quickstart/next-steps.md
+      - quickstart/index.md
+      - quickstart/install.md
+      - quickstart/uiauto2-driver.md
+      - Write a Test:
+          - quickstart/test-js.md
+          - quickstart/test-py.md
+          - quickstart/test-java.md
+          - quickstart/test-rb.md
+      - quickstart/next-steps.md
   - CLI Reference:
-    - cli/index.md
-    - cli/args.md
-    - cli/extensions.md
+      - cli/index.md
+      - cli/args.md
+      - cli/extensions.md
   - Guides:
-    - guides/migrating-1-to-2.md
-    - guides/managing-exts.md
-    - guides/config.md
-    - guides/security.md
-    - guides/caps.md
-    - guides/context.md
-    - guides/settings.md
-    - guides/execute-methods.md
-    - guides/event-timing.md
-    - guides/log-filters.md
-    - guides/grid.md
-    - guides/caching.md
+      - guides/migrating-1-to-2.md
+      - guides/managing-exts.md
+      - guides/config.md
+      - guides/security.md
+      - guides/caps.md
+      - guides/context.md
+      - guides/settings.md
+      - guides/execute-methods.md
+      - guides/event-timing.md
+      - guides/log-filters.md
+      - guides/grid.md
+      - guides/caching.md
   - Developer Reference:
-      - BaseDriver Commands: reference/commands/appium_base_driver-1._appium_base_driver.md
-      - FakeDriver Commands: reference/commands/appium_fake_driver-1._appium_fake_driver.md
+      - BaseDriver Commands: reference/commands/base_driver.md
```

Comment:
should it not be `base-driver`? and same next line?

Reply:
oops